libkbfs/keybase_daemon_rpc.go

```diff
@@ -378,6 +378,7 @@ func (k *KeybaseDaemonRPC) OnConnect(ctx context.Context,
 		keybase1.IdentifyUiProtocol(daemonIdentifyUI{k.daemonLog}),
 		keybase1.NotifySessionProtocol(k),
 		keybase1.NotifyUsersProtocol(k),
+		keybase1.FsProtocol(&fsRPC{config: k.config, log: k.log}),
```

Comment:
That probably means you'll need to add something to `kbfsfuse` and `kbfsdokan` to initialize that `fsRPC` connection, if we want it live for those binaries.  Maybe we only want it on mobile?